pages/juan-ortizc/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Carousel of image</title>
+  </head>
+  <body>
+    <section class="carousel">
+      <header class="sliders">
+        <h1 class="sliders__tittle">
+          OUR <span class="sliders__tittle--orange">HOTELS</span>
+        </h1>
+      </header>
+      <div>
+        <div class="cards-container">
+          <div class="icon">
+            <button class="icon__arrow"><</button>
+          </div>
```

Comment:
Why are you using a `div` as a container?  is `icon` a separate block of carousel?